conf/app.routes

```diff
@@ -2,7 +2,7 @@
 
 ->         /govuk-frontend          govuk.Routes
 ->         /hmrc-frontend           hmrcfrontend.Routes
-GET        /consent-head            uk.gov.hmrc.trackingconsentfrontend.controllers.ConsentPartialController.head
+GET        /tracking.js             controllers.Assets.at(path="/public", file="servicePage.js")
```

Comment:
Should the asset file name match the URL?

Reply:
It probably should really.